### PR TITLE
Add zazu-emoji plugin to plugin list

### PR DESCRIPTION
**Pull request by iansinnott**
**Changes:**

* Added a new markdown file under `docs/_packages` documenting emoji plugin
* Added a png image to serve as the plugins icon

Here's a link to the plugin repo: https://github.com/iansinnott/zazu-emoji

### CODE REVIEW

**Comment by bayleedev** [comment]
Much competition xD

File: docs/_packages/emoji.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+top:         true
+type:        plugin
+title:       "Emoji"
+image:       /images/package-icons/zazu-emoji.png
+githuburl:   iansinnott/zazu-emoji
+description: "⚡ Fast, offline emoji search for Zazu"
```
Comment (bayleedev) [inline docs/_packages/emoji.md]: can you remove `from Zazu` here? Being on zazu's pages it's redundant.

Comment (iansinnott) [reply docs/_packages/emoji.md]: sure

**Comment by iansinnott** [comment]
Yeah I checked out the other emoji plugin as well, but I just wanted one that would work offline. The faster the better for me :D

**Comment by bayleedev** [comment]
Thank you!

**Pull request merged**